db/migrate/20221019143406_add_user_ref_to_post.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] Kindly change the name of the foreign key column name to `author` instead of `user`.